lib/datadog/profiling/ext.rb

```diff
@@ -23,6 +23,8 @@ module HTTP
           FORM_FIELD_TAG_RUNTIME_VERSION = 'runtime_version'
           FORM_FIELD_TAG_SERVICE = 'service'
           FORM_FIELD_TAG_VERSION = 'version'
+          FORM_FIELD_TAG_GIT_REPOSITORY_URL = 'git.repository_url'
```

Comment:
@ivoanjo I am not sure if these constants should be called `FORM_FIELD_*` but others in these file are

Reply:
on the other hand I start to think that constants in Core must not have the `_dd` prefix if it is only for tracing

so in this case I would use Core::Git::Ext::GIT_REPOSITORY_URL for profiling and then append '_dd.' prefix when formatting traces in tracing component

wdyt?

Reply:
@ivoanjo I renamed the constants - now ready for a next pass :)